packages/salesforcedx-utils-vscode/src/commands/notificationService.ts

```diff
@@ -7,10 +7,8 @@
 
 import { Observable } from 'rxjs/Observable';
 import * as vscode from 'vscode';
-
 import { CommandExecution } from '../cli';
 import { nls } from '../messages';
-import { channelService } from './index';
```

Comment:
Similar to the comment on `commands/index.ts` this gets loaded at startup, which forces us to initialize the channel service with the default name. 
I also opted to remove channel service from here since we could still call showing the channel output directly from the command implementation in the extension side.